client/src/components/FormInput/FormInput.tsx

```diff
@@ -23,6 +23,8 @@ interface FormInputProps {
   selectOptions?: Array<{ label: string; value: string }> | undefined;
   selectValue?: string;
 
+  placeholder?: string;
+
```

Comment:
Underneath we are already passing `inputProps` to the interface. This is essentially all inputProps. You can actually remove this and just pass in the `placeholder` property to the `FormInput` and it will work the same way.